index.html

```diff
@@ -1,267 +1,173 @@
+      </section>
+      <section class="section">
+        <h2>Step 1: Create your environment</h2>
+        <p>To set up the DocuSign API Postman Collections on your system, you need a DocuSign developer demo account and <a href="https://www.postman.com/downloads/" target="_blank">Postman</a> installed. If you&rsquo;ve previously set up an account, <strong><a href="https://appdemo.docusign.com/" target="_blank">log in</a></strong>. Otherwise, <strong><a href="https://go.docusign.com/o/sandbox/" target="_blank">create an account</a></strong> for free at the <a href="https://developers.docusign.com/" target="_blank">DocuSign Developer Center</a>.</p>
+        <p>Once you're in your demo account, go to Admin, then choose <a href="https://admindemo.docusign.com/api-integrator-key" target="_blank">API and Keys</a> from the left nav. From here, you'll copy values into the environment setup. You'll need the following values before you click the button below to set up your Postman environment:</p>
+        <ul>
+          <li><strong>Integration key:</strong> Any app you create with DocuSign is called an <em>integration.</em> All your integrations are identified in your DocuSign demo account by their <em>integration key,</em> which is a <a href="http://guid.one/guid" target="_blank">GUID</a> issued to your integration when you select <strong>Add App / Integration Key</strong> on the Apps and Keys page. If you don't have any integrations in your demo account yet, select that button and give your integration a name. Use the integration key value, which is automatically generated when you click <strong>OK.</strong></li>
+          <li><strong>Corresponding secret key:</strong> For Authentication Code Grant authentication, your integration needs a secret key as well. Select Add Secret Key on the Apps and Keys page and copy the value; you can only do this when you create the key, as it won't be visible thereafter.</li>
+          <li><strong>Environment:</strong> Choose Demo.</li>
+          <li><strong>API Version:</strong> The Postman collection supports the eSignature REST API versions 2 and 2.1. Version 2.1 is preferred; version 2 will be deprecated in the future.</li>
+        </ul>
+        <div style="margin: 10px 0">
+          <p><a id="enter-creds" href="#" class="btn btn-primary">Set up your environment</a></p>
+        </div>
+      </section>
+      <section class="section">
+        <h2>Step 2: Import into Postman</h2>
+        <div id="shownEnv" class="">
+          <p>Be sure to complete step 1 before clicking the button below. This imports the DocuSign Postman collection and your environment variables into your installation of Postman.</p>
+        </div>
+        <div id="hiddenEnv" class="nodisplay">
+          <p>
+          <div class="postman-run-button"
+            data-postman-action="collection/import"
+            data-postman-var-1="8624faf20177f06bd69f"></div>
```

Comment:
This uuID should not change, unless we are uploading a new postman collection

Reply:
Is there a way to not have this UUID not be specified?